src/components/Tool/Appeal/AddAppealForm.tsx

```diff
@@ -105,7 +154,32 @@ const AddAppealForm = (): ReactElement => {
     name: yup.string().required('Please enter a name'),
   });
 
-  return (
+  const contactStatusesFormatted = Object.entries(
+    contactTags,
+  ).map(([tag, translatedTag]) => ({ value: tag, title: translatedTag }));
+
+  const contactTagsList = data?.accountList.contactTagList ?? [];
+
+  const selectAllStatuses = (): void => {
+    setFilterTags((prevState) => ({
+      ...prevState,
+      statuses: contactStatusesFormatted.slice(
+        0,
+        contactStatusesFormatted.length - 2,
+      ),
```

Comment:
What is this doing? Removing the last status? Why? I'm just don't have the context here.

Reply:
Oh yeah the "Select All" option doesn't actually select all since I think there are contradicting options ("all active" and "all hidden"). I moved them to the end to make it easier and just cut them off.

Reply:
Even before mentioning loading them from the API, I was going to suggest filtering them out by id/name. I think it's confusing to read the code and error prone to rely on indexes and sort order if future changes are made to the list.